api/bootstrap.proto

```diff
@@ -220,11 +220,24 @@ message StatsdSink {
   }
 }
 
+// Stats configuration proto schema for built-in *envoy.dog_statsd* sink.
+// The sink emits stats with `DogStatsD <https://docs.datadoghq.com/guides/dogstatsd/>`_
+// compatible tags. Tags are configurable via :ref:`StatsConfig <envoy_api_msg_StatsConfig>`.
+message DogStatsdSink {
+  // The UDP address of a running DogStatsD compliant listener. If specified,
```

Comment:
Does datadog support TCP also?

Reply:
It seems dd-agent (the datadog's agent) doesn't support TCP: https://github.com/DataDog/dd-agent/wiki/Agent-Architecture#supervision-privileges-and-network-ports

statsd_exporter does: https://github.com/prometheus/statsd_exporter#building-and-running